sensehat/README.md

```diff
@@ -54,6 +54,19 @@ display.draw(bitmap);
 // Close the display when done.
 display.close();
 ```
+```
```

Comment:
You can add Java after the backticks to add code highlighting to the code block. Also, you won't need to add additional indenting.

```java
// just a snippet that verifies the device is working
// your actual use would of course be much more awesome
try (BaroTemp bt = SenseHat.openBaroTemp()) {
    bt.setBarometerOffset(-6.2);
    Log.i(TAG, "PressureRaw: " + bt.getBarometerRaw());
    Log.i(TAG, "TemperatureRaw: " + bt.getTemperatureRaw());
    Log.i(TAG, "Pressure: " + bt.getBarometer());
    Log.i(TAG, "Temperature: " + bt.getTemperature());
} catch (Exception e) {
    Log.e(TAG, "Oops", e);
}
```